### ISSUE TITLE
sometimes realtime information is missing even if id realtime is set to "1" for certain departures. This crashes the app.

### ISSUE THREAD

**Issue by ghost91-** [fsimphy/bayernfahrplan#2]
In gitlab by @ghost on May 26, 2017, 17:50

## Summary ##

Sometimes certain departures contain ```<realtime>1</realtime>```, but the ```rt``` tag is missing. This makes the app crash.

### Expected behaviour ###
The app should display the panned departure time, if rt info is missing.

### Actual behaviour ###
The app crashes.

## Example XML ##
```
<dp>
<n>Regensburg Universität</n>
<realtime>1</realtime>
<dt>
<da>26.5.2017</da>
<t>17:49</t>
</dt>
<st>
<da>20170526</da>
<t>1749</t>
</st>
<m>
<n>RVV-Bus-Linie</n>
<nu>6</nu>
<ty>5</ty>
<co>3</co>
<des>Wernerwerkstr.</des>
<dy>-9999</dy>
<dv>
<branch>11</branch>
<li>006</li>
<su>A</su>
<pr>6aa</pr>
<dct>R</dct>
<ne>rvv</ne>
<tk>308</tk>
<stateless>308</stateless>
</dv>
</m>
<r>
<id>4014080</id>
<a>0</a>
<pl>Bstg. E</pl>
</r>
<c>12092352.84663,48998735.63537</c>
<ns/>
<nx/>
</dp>
```

**Comment by ghost91-** [comment]
In gitlab by @ghost on Jul 12, 2017, 15:43

Mentioned in merge request !4

**Comment by ghost91-** [comment]
In gitlab by @ghost on Jul 12, 2017, 15:43

Status changed to closed by commit 7a76d12ec2db4a6ec68dff83a94883287f39f132

**Comment by ghost91-** [comment]
In gitlab by @ghost on Jul 12, 2017, 15:54

Mentioned in commit 7a76d12ec2db4a6ec68dff83a94883287f39f132
